Tutorial.md

```diff
@@ -11,477 +10,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+```
 
-    FORMAT: 1A
+## Metadata
 
-Depending on the tools you will be using with your blueprint you might want to start your blueprint file with a metadata explicitly stating the API Blueprint version you want to use like in this case the version 1A.
+The blueprint starts wih a metadata section. In this case we have specified that `FORMAT` has the value of `1A`. The format keyword is used to specify the version of the API Blueprint. In this case 1A.
 
-> **Note:** Should you need a clarification of some terms as used through this document refer to [API Blueprint Glossary of Terms][].
-
-#### API Name
-
-    # Gist Fox API
-
-Every good API should have a name. So does ours – *"Gist Fox API"*. The first Markdown header in our document represents just that – the API name.
-
-#### API Description
-
-    Gist Fox API is a **pastes service** similar to [GitHub's Gist][http://gist.github.com].
-
-The API Name might be followed by any arbitrary Markdown-formatted discussion. Preferably about your API.
-
-> **Note:** We will add much richer and more helpful discussion in the final version of this blueprint. Adding details about authentication, used media types and error handling to API description is always a great idea!
->
-> **Note:** For the sake of simplicity we will now omit user management and authentication. These features will be discussed in a future instalment of the API Blueprint Tutorial.
-
-## Markdown
-All you really need to write a blueprint is a text editor. A Markdown editor is even better. Anything you like. From Vi to Byword. Online editors are great too! You can even directly edit a blueprint on GitHub in the repository your service is living!
-
-If you are completely new to Markdown it is now the best time to learn about it. Just head over to [Markdown Tutorial][] and enjoy the ride. Just don't forget to come back we have some cool stuff too!
-
-Should you just need to refresh Markdown syntax you can always check the one and only [Gruber's Original][] documentation.
-
->  **Note:** It is not essentially required to know the Markdown syntax but it will help you a lot. The knowledge of Markdown will make some API Blueprint constructs more intuitive.
-
-## First Resource
-The first resource in our API is its root. The entry point to our API is defined like so:
-
-    # Gist Fox API Root [/]
-    Gist Fox API entry point.
-
-    This resource does not have any attributes. Instead it offers the initial API affordances in the form of the HTTP Link header and HAL links.
-
-    ## Retrieve Entry Point [GET]
+## API Name & Description
 
-    + Response 200 (application/hal+json)
-        + Headers
+The first heading in the blueprint serves as the name of your API, which in this case is "Polls". Headings start with one or more `#` symbols followed by a title. The API Name here uses one hash to distinguish it as the first level. The number of `#` you use will determine the size of the heading.
```

Comment:
Is it the "size of the heading" or is it the level of the heading (or something similar)?